src/test/kotlin/uk/gov/justice/digital/hmpps/hmppscontactsapi/integration/resource/ReferenceCodesResourceIntegrationTest.kt

```
@@ -45,13 +48,23 @@ class ReferenceCodesResourceIntegrationTest : H2IntegrationTestBase() {
 
   @Test
   fun `should return empty list if no matching code found`() {
```

this method name should be updated to 'fun `should return an error if no matching code is found`?